compose/config/config.py

```diff
@@ -257,13 +257,20 @@ def with_abs_paths(cls, working_dir, filename, name, config):
             config)
 
 
-def find(base_dir, filenames, environment, override_dir=None):
+def find(base_dir, filenames, environment, override_dir=None, url=None):
     if filenames == ['-']:
         return ConfigDetails(
             os.path.abspath(override_dir) if override_dir else os.getcwd(),
             [ConfigFile(None, yaml.safe_load(sys.stdin))],
             environment
         )
+    if url:
+        response = six.moves.urllib.request.urlopen(url)
```

Comment:
A lot could go wrong here so probably need some error handling somewhere. Not sure where this would propagate up to currently.